packages/components/src/Components/NotAuthorized/NotAuthorized.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Title,
+  Button,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateIcon,
+  EmptyStateBody
+} from '@patternfly/react-core';
+
+import { LockIcon } from '@patternfly/react-icons';
+import { PageHeader, PageHeaderTitle, Main } from '@redhat-cloud-services/frontend-components';
+
+import './NotAuthorized.scss';
+
+const NotAuthorized = ({ serviceName }) => {
+  return (
+    <React.Fragment>
+      <EmptyState variant={ EmptyStateVariant.full } className='ins-c-not-authorized'>
+        <EmptyStateIcon icon={ LockIcon } />
+        <Title headingLevel="h5" size="lg"> You do not have access to { serviceName }</Title>
+        <EmptyStateBody>
+          Contact your organization administrator(s) for more information.
+        </EmptyStateBody>
+        {
+          document.referrer ?
+            <Button variant="primary" onClick={ () => history.back() }>Return to previous page</Button> :
```

Comment:
Just a nitpick, I think what @karelhala mentioned before was that it'd be good to have a `history` prop you could pass to this component to go back in history within the app without having to reload

Reply:
@dLobatog This is what you originally requested (using the browser's `history`), and that's what rbac-ui does. It seems to work pretty well - try to navigate to edit user roles in CI with your account, and you'll see this work as written here in the rbac-ui. https://github.com/RedHatInsights/insights-rbac-ui/blob/master/src/presentational-components/states/DeniedState.js#L32 No React required! What do you think?